packages/commonwealth/client/scripts/models/Comment.ts

```diff
@@ -21,6 +21,10 @@ class Comment<T extends IUniqueId> {
   public readonly lastEdited: moment.Moment;
   public readonly deleted: boolean;
 
+  public readonly canvasAction: string;
```

Comment:
Can we make these an object, like `canvasMeta`, rather than having 3 properties per model?